File: 01_BasicPagramming/Lab02/Python_Lang/Exercise01/Demo01.py
def myInput():
    n = int(input("Enter n = "))
    return n

def sum(x, y):
    return x + y

def myOutput(a, b, c):
    print("sum = %d + %d = %d" %(a, b, c))

def main():
    a = myInput()
    b = myInput()
    s = sum(a, b)
    myOutput(a, b, s)

if __name__ == "__main__":
    main()

File: Session_01_BasicPagramming/Lab02/C_Lang/Exercise01/src/Demo01.cpp
#include <stdio.h>

int input()
{
    int n;
    printf("Enter n = ");
    scanf("%d", &n);
    return n;
}

int Sum(int a, int b)
{
    int sum = a + b;
    return sum;
}

void output(int a, int b, int c)
{
    printf("s = %d + %d = %d", a, b, c);
}

int main()
{
    int a; 
    int b;
    int sum; 

    // Input
    a = input();
    b = input();

    // Calculate
    sum = Sum(a, b);

    // Out
    output(a, b, sum);

    return 0;
}